src/lib/ledger_catchup/ledger_catchup.ml

```diff
@@ -289,19 +182,25 @@ module Make (Inputs : Inputs.S) :
 
   let run ~logger ~network ~frontier ~catchup_job_reader
       ~catchup_breadcrumbs_writer ~unprocessed_transition_cache =
-    Strict_pipe.Reader.iter catchup_job_reader ~f:(fun subtree ->
-        match%bind
-          get_transitions_and_compute_breadcrumbs ~logger ~network ~frontier
-            ~num_peers:8 ~unprocessed_transition_cache ~target_subtree:subtree
-        with
-        | Ok tree -> Strict_pipe.Writer.write catchup_breadcrumbs_writer [tree]
+    Strict_pipe.Reader.iter catchup_job_reader ~f:(fun forest ->
```

Comment:
Can you write this explicitly as a tuple? I thought this variable was just a forest, but it is a forest with transition